internal/utils/config.go

```diff
@@ -181,23 +181,23 @@ var Config = config{
 			Template: "Your code is {{ .Code }} .",
 		},
 		External: map[string]provider{
-			"apple":     {},
-			"azure":     {},
-			"bitbucket": {},
-			"discord":   {},
-			"facebook":  {},
-			"github":    {},
-			"gitlab":    {},
-			"google":    {},
-			"keycloak":  {},
-			"linkedin":  {},
-			"notion":    {},
-			"twitch":    {},
-			"twitter":   {},
-			"slack":     {},
-			"spotify":   {},
-			"workos":    {},
-			"zoom":      {},
+			"apple":         {},
+			"azure":         {},
+			"bitbucket":     {},
+			"discord":       {},
+			"facebook":      {},
+			"github":        {},
+			"gitlab":        {},
+			"google":        {},
+			"keycloak":      {},
+			"linkedin_oidc": {},
```

Comment:
```suggestion
			"linkedin":          {}, // TODO:
			"linkedin_oidc": {},
```

Technically this is a breaking change for config.toml that already uses `linkedin` provider. Let's keep the old field but make it a noop so we don't break users existing ci pipelines.